exec/Chainweb/Gaps.hs

```diff
@@ -215,32 +186,27 @@ withDroppedIndexes pool logger action = do
     fmap fst $ generalBracket (dropIndexes pool indexInfos) release (const action)
   where
     release _ = \case
-      ExitCaseSuccess _ -> dedupeTables pool logger
+      ExitCaseSuccess _ -> return () -- dedupeTables pool logger
```

Comment:
This needs to change before merging.  It's just here temporarily to allow me to do more experiments.